src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java

```diff
@@ -514,16 +519,14 @@ public Integer getVideoWidth(WebElement thumbnail) {
     return videoWidth;
   }
 
-  public void verifyVideoWidth(int widthDesired) {
+  public int getVideoWidth() {
     int videoWidth = getVideoWidth(videoThumbnail);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "videoWidth". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1488)